website/common/script/content/spells.js

```diff
@@ -452,7 +450,6 @@ spells.special = {
       if (!target.items.special.greetingReceived) target.items.special.greetingReceived = [];
       target.items.special.greetingReceived.push(user.profile.name);
 
-      if (!target.flags) target.flags = {};
```

Comment:
I think this is the source of the error I'm hitting. If the target doesn't have the `flags` object, we hit an error when trying to set a property on it. Why were these removed?